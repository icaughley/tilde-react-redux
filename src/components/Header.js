import React from "react";
import "../style/style.css";
import logo from "../images/logo.gif";
import stocksoftware from "../images/stocksoftware.gif";

export default function Header() {
    return (
        <div className="page_header">
            <a href="http://www.stocksoftware.com.au"><img src={stocksoftware} alt="Stock Software"/></a>
            <img className="logo" src={logo} alt="logo"/>
        </div>
    );
}