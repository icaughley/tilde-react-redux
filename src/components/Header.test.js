import React from "react";
import renderer from "react-test-renderer";
import {shallow} from "enzyme";
import Header from "./Header";

describe('Header component', () => {
    it('has not changed', () => {
        expect(renderer.create(<Header />)).toMatchSnapshot();
    });

    it('has the correct class', () => {
        const component = shallow(<Header />);
        expect(component.hasClass('page-header')).toEqual(true);
    });
});
