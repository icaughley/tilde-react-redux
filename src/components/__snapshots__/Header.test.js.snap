exports[`Header component has not changed 1`] = `
<div
  className="page-header">
  <a
    href="http://www.stocksoftware.com.au">
    <img
      alt="Stock Software"
      src="stocksoftware.gif" />
  </a>
  <img
    alt="logo"
    className="logo"
    src="logo.gif" />
</div>
`;
